File: auriculoterapia_api/Auriculoterapia.Domain/Tratamiento.cs
using System;

namespace Auriculoterapia.Domain
{
    public class Tratamiento
    {
        public int Id {get; set;}
        public string TipoTratamiento {get; set;}
        public Date FechaInicio {get;set;}
        public Date FechaFin {get; set;}
        public int FrecuenciaAlDia {get; set;}
        public int TiempoPorTerapia {get; set;}
        

        public int SolicitudTratamientoId {get; set;}
        public SolicitudTratamiento SolicitudTratamiento {get; set;}
        
    }
}

File: Auriculoterapia/app/src/main/java/com/example/android/auriculoterapia_app/models/Tratamiento.kt
package com.example.android.auriculoterapia_app.models

import java.io.Serializable

data class Tratamiento(
    var id: Int,
    var tipoTratamiento: String,
    var fechaEnvio: String,
    var fechaInicio: String,
    var fechaFin: String,
    var frecuenciaAlDia: Int,
    var tiempoPorTerapia: Int,
    var imagenEditada: String,
    var solicitudTratamientoId: Int,
    var solicitudTratamiento: SolicitudTratamiento,
    var estado: String
): Serializable{

}